src/GameObject.h

```diff
@@ -2,12 +2,13 @@
 #include "IDrawable.h"
 #include <GL/freeglut.h>
 
-class GameObject: public IDrawable {
+class GameObject : public IDrawable {
 private:
 	float x, y;
 	float width, height;
 public:
 	GameObject(float, float);
 	void draw();
+	void keyboard_down(unsigned char, int, int);
+	
```

Comment:
Please move the `keyboard_down` function to the main and add four additional function to GameObject:
* moveUP()
* moveDown()
* moveRight()
* moveLeft()
And call these functions in the main's `keyboard_down` function.